buildScript/webpack.config.js

```diff
@@ -183,6 +183,11 @@ export default function makeWebpackConfig(config) {
         );
     }
 
+    if (!ENV_DEV_MODE) { // Adding this so we see it in the log file of our builds
+        console.log('Building client with Global Props:');
+        console.log(globals.__PROPS__);
+    }
+
```

Comment:
In the need future, we should start adding a `-verbose` flag.  The additional details is useful, but makes it harder to identify warnings and errors from build.

Reply:
Makes sense.